config/locales/views/groups/en.yml

```diff
@@ -25,6 +26,7 @@ en:
     due_date_extension: Due Date Extension
     duration_extension: Duration Extension
     empty: Empty Group
+    exam_template: Exam Template
```

Comment:
Don't repeat this, you can reuse an existing string. In views, you aren't restricted to only use keys that are in the same namespace (e.g., group); you can use other keys as well. This is preferable to duplicating them.